src/runtime/virtcontainers/network.go

```diff
@@ -271,7 +276,7 @@ func generateVCNetworkStructures(ctx context.Context, endpoints []Endpoint) ([]*
 			Type:        string(endpoint.Type()),
 			RawFlags:    noarp,
 			HwAddr:      endpoint.HardwareAddr(),
-			PciPath:     endpoint.PciPath().String(),
+			PciPath:     pciPath,
```

Comment:
It is confusing to  reuse the PciPath field for non-PCI stuff. Can you please rename the field to something more abstract, e.g. DevicePath ?